mock/src/test/java/com/tngtech/keycloakmock/api/TokenConfigTest.java

```diff
@@ -301,6 +301,7 @@ void unexpected_issuer_causes_exception() {
     Builder builder = aTokenConfig();
     assertThatThrownBy(() -> builder.withSourceToken(TOKEN_WITH_UNEXPECTED_ISSUER_URL))
         .isInstanceOf(IllegalArgumentException.class)
-        .hasMessageContaining("did not conform to the expected format");
+        .hasMessageContaining("did not conform to the expected format")
+        .hasMessageContaining("'http[s]://$HOSTNAME[:port][:contextPath]/realms/$REALM'");
```

Comment:
The colon in front of `port` was not meant as an indicator for parameter replacement but as part of the string, e.g. ":8080". Thus, I'd prefix the context path with a slash instead. To make it more clear, I think we can also use the same variable replacement syntax as for the hostname and the realm:
```suggestion
        .hasMessageContaining("'http[s]://$HOSTNAME[:$PORT][/$CONTEXT_PATH]/realms/$REALM'");
```

Reply:
Thanks for the explanation, now this format makes more sense.